### ISSUE TITLE
README / docs

### ISSUE THREAD

**Issue by JRMeyer** [dabinat/bad-cv-clips#1]
it's not obvious what the `csv` file contains for transcripts... are these the "good" transcripts you have made yourself? what's the scope of this list? what is a bad utterance?

I have some bad clips I've come across, but not sure if they are the same kind of "bad"

**Comment by dabinat** [comment]
I've updated the readme with more info - let me know if anything is still unclear.

**Comment by JRMeyer** [comment]
`README` is now more clear, but now I wonder if grouping all these different kinds of "bad" together is a good idea

If the transcript is incorrect, I agree excluding that example makes sense, but (for example) someond speaking in a quiet voice seems fine to include in ASR training

This list is long already, but it could be very useful to have information on why each clip is "bad", so that you could selectively leave out examples

**Comment by dabinat** [comment]
Yeah, I did think of that but only after I’d already built up a list of several hundred. But to be honest, I don’t really know why you would filter any of them out. The list is intended to be a pretty objective selection of clips that shouldn’t have made it through validation. Many of them are egregiously bad examples that should be immediately obvious.

When I say quiet clips, I mean clips that are so quiet that the speech isn’t even visible on the waveform. Remember that DeepSpeech was used in the process of flagging these up, which gave me a good general idea of what volume levels are workable. DeepSpeech can indeed work with very quiet audio, but in a lot of these cases the transcript came back completely blank because it didn’t detect that there was speech even in the file.

I did anticipate that there may be some skepticism, which is why the original transcript is included in the file. It allows you to easily do random spot checks on files in the list - just copy and paste the filename into ffplay.
